src/Components/Loading.js

```diff
@@ -0,0 +1,17 @@
+import CloseButton from "./CloseButton";
+
```

Comment:
I love this loading graphic! It was only there for a brief moment on my screen but it was a nice touch.